components/ActivityIndicator/src/MDCActivityIndicator.h

```diff
@@ -97,8 +97,8 @@ IB_DESIGNABLE
 
 /**
  The array of colors that are cycled through when animating the spinner. Populated with default
- colors of blue, red, yellow and green on initialization. An empty array results in a blue spinner
- with no color cycling.
+ colors of blue, red, yellow and green on initialization. Passing an empty array will cause the
```

Comment:
A suggestion: "If an empty array is provided to this property's setter, then the provided array will be discarded and an array consisting of the default color values will be assigned instead."